### PR TITLE
[JENKINS-36720] - Fix SpotBugs warnings for "known as null" values

### PR DESCRIPTION
**Pull request by StefanSpieker**
Replaced a known null value in a string with a non null more reasonable value and replaced a known as null value with null. Both were findings by Spotbugs.

### Proposed changelog entries

* Internal: Fixed spotbugs issues

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

File: core/src/main/java/jenkins/util/SystemProperties.java [hunk 1]
```diff
@@ -199,7 +199,7 @@ public static String getString(String key) {
         }
         
         if (LOGGER.isLoggable(Level.CONFIG)) {
-            LOGGER.log(Level.CONFIG, "Property (not found): {0} => {1}", new Object[] {key, value});
+            LOGGER.log(Level.CONFIG, "Property (not found): {0} => {1}", new Object[] {key, null});
```
Comment (varyvol) [inline core/src/main/java/jenkins/util/SystemProperties.java]: I think it's better to change the message, it makes no sense to print something we know that won't have a value.

File: core/src/main/java/hudson/model/DownloadService.java [hunk 2]
```diff
@@ -401,7 +401,7 @@ public FormValidation updateNow() throws IOException {
             for (UpdateSite updatesite : Jenkins.getActiveInstance().getUpdateCenter().getSiteList()) {
                 String site = updatesite.getMetadataUrlForDownloadable(url);
                 if (site == null) {
-                    return FormValidation.warning("The update site " + site + " does not look like an update center");
+                    return FormValidation.warning("The update site " + updatesite + " does not look like an update center");
```
Comment (fcojfernandez) [inline core/src/main/java/hudson/model/DownloadService.java]: Could you please show us an example of the resulting warning?

Comment (varyvol) [reply core/src/main/java/hudson/model/DownloadService.java]: Maybe updatesite.getId() could be used?

File: core/src/main/java/jenkins/util/SystemProperties.java (hunk 1 above)
Comment (StefanSpieker) [reply core/src/main/java/jenkins/util/SystemProperties.java]: You are right. I changed it.

File: core/src/main/java/hudson/model/DownloadService.java (hunk 2 above)
Comment (StefanSpieker) [reply core/src/main/java/hudson/model/DownloadService.java]: Thanks for the suggestion. I used getId() instead.

**Review by timja — APPROVED** [review]

**Review by fcojfernandez — APPROVED** [review]

**Review by varyvol — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Looks good, thanks!
I plan to merge it tomorrow if no negative feedback

**Pull request merged**